src/stores/calls/private-call/reducer.ts

```diff
@@ -28,6 +29,7 @@ export type PrivateCallState = {
   newCall: null | NewCall;
   newStream: null | NewStream;
   newInjectionStream: null | NewInjectionStream;
+  activeInjectionMute: boolean;
```

Comment:
**suggestion(blocking)**: Let's rename the variable. The muted/unmuted status is related to the bot, not to the injection, so I suggest changing it to something like _isBotMuted_

Reply:
Hi, your suggestion has already been applied. Thanks